probe/docker/registry.go

```diff
@@ -472,13 +472,26 @@ func (r *registry) WalkNetworks(f func(docker_client.Network)) {
 	}
 }
 
-// ImageNameWithoutVersion splits the image name apart, returning the name
-// without the version, if possible
-func ImageNameWithoutVersion(name string) string {
+// ImageNameParts returns parts of the full image name (image name, image tag).
```

Comment:
nit: I understand that these functions were here before, but it makes me think if `container.go` would not be a better place for them, WDYT?